services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -229,6 +239,46 @@ export class CollectionFlowService {
     return flowData;
   }
 
+  private async _persistFileUrlsToDocuments(
```

Comment:
Note the additional `_` for private.

```suggestion
  private async __persistFileUrlsToDocuments(
```